runltp

```diff
@@ -78,7 +78,7 @@ setup()
     }
     export LTPROOT=${PWD}
     export TMPBASE="/tmp"
-    export PATH="${PATH}:${LTPROOT}/testcases/bin:${LTPROOT}/bin"
+    export PATH="${PATH}:${LTPROOT}/testcases/bin:${LTPROOT}/bin"$( find ${LTPROOT}/testcases/bin/ddt -type d -exec printf {}":" \; )
```

Comment:
I don't think that it's reasonable to clutter the $PATH with all subdirectories with executables in ddt directory.